src/TTGamesExplorerRebirthUI/Helper.cs

```diff
@@ -146,19 +169,24 @@ public static void OpenFileInternal(string gameFolderPath, string path, byte[] f
                     {
                         fileBuffer ??= File.ReadAllBytes(path);
 
-                        new SoundForm(path, fileBuffer).ShowDialog();
+                        form = new SoundForm(path, fileBuffer);
                         break;
                     }
 
                 case ".pc_shaders":
                     {
                         fileBuffer ??= File.ReadAllBytes(path);
 
-                        new PCShadersForm(path, fileBuffer).ShowDialog();
+                        form = new PCShadersForm(path, fileBuffer);
                         break;
 
                     }
             }
+            if (form != null)
+            {
+                form.ShowDialog();
+                GC.Collect(GC.MaxGeneration, GCCollectionMode.Aggressive);
+            }
```

Comment:
Calling `GC.Collect()` here isn't really needed, maybe you should revert those changes.